lms/static/sass/features/_course-experience.scss

```diff
@@ -171,3 +171,21 @@
     }
   }
 }
+
+// Course Updates Page
+div.course-updates {
+  div.all-updates {
+      article.updates-article {
+        margin: 10px 0px;
+        .date {
+          font-size: 14px;
```

Comment:
To support theming, we don't specify font sizes. You should use a function like this:

    font-size(medium);